bridge/src/wrapped-ncg-minter.ts

```diff
@@ -37,6 +38,7 @@ export class WrappedNCGMinter implements IWrappedNCGMinter {
       const gasPriceString = await this._web3.eth.getGasPrice();
       const gasPrice = new Decimal(gasPriceString);
       const gasPriceWithTip = gasPrice.mul(this._gasTipRatio).floor();
-        return this._contract.methods.mint(address, this._web3.utils.toBN(amount.toString())).send({from: this._minterAddress, gasPrice: gasPriceWithTip.toString()});
+      const promiEvent: PromiEvent<TransactionReceipt> = this._contract.methods.mint(address, this._web3.utils.toBN(amount.toString())).send({from: this._minterAddress, gasPrice: gasPriceWithTip.toString()});
+        return promiEvent.on("transactionHash", transactionHash => console.log("Transaction Hash is", transactionHash));
```

Comment:
```suggestion
      return promiEvent.on("transactionHash", transactionHash => console.log("Transaction Hash is", transactionHash));
```
Does any meaning of indent here?

Reply:
No, I will remove the indent 👍🏻 

P.S. In fact, other lines' indent is incorrect 😅